### PR TITLE
Add Documentation for metrics

### PR DESCRIPTION
**Pull request by jcscottiii**

### CODE REVIEW

File: content/ops/metrics.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+title: Metrics
+weight: 10
+---
+
+# Requesting an account on Metrics
+1. Please [create a ticket](https://github.com/18f/devops/issues/new) with the title including `[Metrics Acct Req]`
+1. Please indicate if you need an `admin` account or `regular` account.
+
+# Updating the Metrics containers.
+1. SSH into jumpbox
```
Comment (jessieay) [inline content/ops/metrics.md]: instructions or link on how to do this?

File: content/ops/metrics.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+1. Run `docker-compose build && docker-compose up -d`
+1. Verify none of the components went down by running `docker-compose ps`
+
+### Warning: Any commits that update the version of Grafana will wipe out dashboards and users for now. The following actions should be taken in case.
+
+# Pointing Grafana to new Datasource or Updating Datasoruce (Grafana admins)
+1. Currently, the dashboards are looking for the `Name` to be `cf_prd`, `cf_np`, `bosh_prd`, or `bosh_np`. So if you are updating one of those, please ensure the name fits exactly.
+1. Set the `Type` to `InfluxDB 0.9.x`
+1. Given the AWS Internal IP of the Docker Host (Please refer to the AWS EC2 Console), Set the `Url` to `http://<internal-ip>:8086`
+1. Set the `Database`, `User`, and `Password` as appropriate.
```
Comment (jessieay) [inline content/ops/metrics.md:39]: instructions or link on how to do this? (probably obvious to some, but not to me)

File: content/ops/metrics.md (hunk 1 above)
Comment (jcscottiii) [inline content/ops/metrics.md]: Done :)

File: content/ops/metrics.md (hunk 2 above)
Comment (jcscottiii) [inline content/ops/metrics.md:39]: Done :)

**Comment by jcscottiii** [comment]
@jessieay added more details!

**Pull request merged**